stable/etcd-operator/README.md

```diff
@@ -66,6 +66,7 @@ The following table lists the configurable parameters of the etcd-operator chart
 | `deployments.etcdOperator`                        | Deploy the etcd cluster operator                                     | `true`                                         |
 | `deployments.backupOperator`                      | Deploy the etcd backup operator                                      | `true`                                         |
 | `deployments.restoreOperator`                     | Deploy the etcd restore operator                                     | `true`                                         |
+| `customResources.createDefinition`                | Create custom resource definition                                    | `false`                                        |
```

Comment:
I'm not too familiar with this project/chart yet. I see `EtcdCluster` definition, but where are the other CRDs (`EtcdBackup`, `EtcdRestore`) defined?

I had a bit of time to look at https://github.com/coreos/etcd-operator/blob/master/cmd/backup-operator/main.go and https://github.com/coreos/etcd-operator/blob/master/cmd/restore-operator/main.go but haven't grokked the full topology of this operator yet.

Reply:
`createDefinition` would create CRD definition, and `EtcdXx` would create custom resource.
I think that definition must place under `crds` dir.

Reply:
Each controller (etcdbackup and etcdrestore) will automatically create the crds when deployed.